fabtests/functional/rdm_rma_trigger.c

```diff
@@ -75,12 +76,19 @@ static int run_test(void)
 	if (ret)
 		return ret;
 
+	//consume preposted buffer to align the rx_seq with the counter values
```

Comment:
I don't understand this comment, particularly before trying to send a message.

Reply:
The ft_init_fabric call preposts a rx buffer as part of the initialization and in doing so increases the rx_seq which tracks the number of posted receives so this was intended to get rid of that preposted rx buffer so that we could use the rx_seq. But I forgot the rx_cq_cntr value existed which tracks the numbers of received cq/cntr operations so that would make more sense to use anyway. I'll change it.

Reply:
Nevermind, I guess I don't understand what tx_cq_cntr and rx_cq_cntr are because they are always 0. Updated to just read the actual counter value.